### ISSUE TITLE
Weird error on likeAvirgin

### ISSUE THREAD

**Issue by xbill82** [kuzzleio/kuzzle#393]
Don't really have the time to properly check this but I get this error when launching

    bin/kuzzle likeAvirgin

(don't get better if I specify a `--fixtures` file...)

```
{ message: 'Failed to execute CLI action: Bulk import: Parse error: input paramter <bulkData> is missing',
  stack: 'Error\n    at BadRequestError.KuzzleError (/var/app/node_modules/kuzzle-common-objects/errors/kuzzleError.js:12:19)\n    at new BadRequestError (/var/app/node_modules/kuzzle-common-objects/errors/badRequestError.js:6:15)\n    at ElasticSearch.getInfos.client.info.then.res.then.search.requestObject.get.mget.requestObject.count.create.requestObject.createOrReplace.requestObject.update.requestObject.replace.requestObject.delete.requestObject.deleteByQuery.getAllIdsFromQuery.call.then.Promise.createCollection.truncateCollection.import.optionalAttributes.forEach.bulkData.body.forEach.client.bulk.then.response.then.Object.keys.forEach [as import] (/var/app/lib/services/elasticsearch.js:380:29)\n    at /var/app/lib/api/controllers/bulkController.js:19:29\n    at tryCatcher (/var/app/node_modules/bluebird/js/release/util.js:16:23)\n    at Promise._settlePromiseFromHandler (/var/app/node_modules/bluebird/js/release/promise.js:510:31)\n    at Promise._settlePromise (/var/app/node_modules/bluebird/js/release/promise.js:567:18)\n    at Promise._settlePromise0 (/var/app/node_modules/bluebird/js/release/promise.js:612:10)\n    at Promise._settlePromises (/var/app/node_modules/bluebird/js/release/promise.js:691:18)\n    at Async._drainQueue (/var/app/node_modules/bluebird/js/release/async.js:138:16)\n    at Async._drainQueues (/var/app/node_modules/bluebird/js/release/async.js:148:10)\n    at Immediate.Async.drainQueues [as _onImmediate] (/var/app/node_modules/bluebird/js/release/async.js:17:14)\n    at processImmediate [as _immediateCallback] (timers.js:383:17)',
  _source: { body: { fixtures: '/fixtures.json', mappings: '/mappings.json' } } }
{ message: 'Failed to execute CLI action: Bulk import: Parse error: input paramter <bulkData> is missing',
  stack: 'Error\n    at BadRequestError.KuzzleError (/var/app/node_modules/kuzzle-common-objects/errors/kuzzleError.js:12:19)\n    at new BadRequestError (/var/app/node_modules/kuzzle-common-objects/errors/badRequestError.js:6:15)\n    at ElasticSearch.getInfos.client.info.then.res.then.search.requestObject.get.mget.requestObject.count.create.requestObject.createOrReplace.requestObject.update.requestObject.replace.requestObject.delete.requestObject.deleteByQuery.getAllIdsFromQuery.call.then.Promise.createCollection.truncateCollection.import.optionalAttributes.forEach.bulkData.body.forEach.client.bulk.then.response.then.Object.keys.forEach [as import] (/var/app/lib/services/elasticsearch.js:380:29)\n    at /var/app/lib/api/controllers/bulkController.js:19:29\n    at tryCatcher (/var/app/node_modules/bluebird/js/release/util.js:16:23)\n    at Promise._settlePromiseFromHandler (/var/app/node_modules/bluebird/js/release/promise.js:510:31)\n    at Promise._settlePromise (/var/app/node_modules/bluebird/js/release/promise.js:567:18)\n    at Promise._settlePromise0 (/var/app/node_modules/bluebird/js/release/promise.js:612:10)\n    at Promise._settlePromises (/var/app/node_modules/bluebird/js/release/promise.js:691:18)\n    at Async._drainQueue (/var/app/node_modules/bluebird/js/release/async.js:138:16)\n    at Async._drainQueues (/var/app/node_modules/bluebird/js/release/async.js:148:10)\n    at Immediate.Async.drainQueues [as _onImmediate] (/var/app/node_modules/bluebird/js/release/async.js:17:14)\n    at processImmediate [as _immediateCallback] (timers.js:383:17)',
  _source: { body: { fixtures: '/fixtures.json', mappings: '/mappings.json' } } }
```

**Comment by xbill82** [comment]
Closing this issue since I am unable to reproduce it in the current implementation (on `develop` after #403  is merged). The error is different, so I'll create a new issue.
